go.mod

```diff
@@ -5,61 +5,42 @@ go 1.12
 require (
 	github.com/container-storage-interface/spec v1.2.0
 	github.com/golang/mock v1.4.3
-	github.com/golang/protobuf v1.3.5
-	github.com/google/go-cmp v0.4.0 // indirect
+	github.com/golang/protobuf v1.4.2
 	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/kubernetes-csi/csi-lib-utils v0.7.0
 	github.com/kubernetes-csi/csi-test v2.0.0+incompatible
-	github.com/prometheus/client_golang v1.0.0
+	github.com/prometheus/client_golang v1.7.1
 	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.4.1
+	github.com/prometheus/common v0.10.0
 	google.golang.org/grpc v1.28.0
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/client-go v0.18.0
-	k8s.io/code-generator v0.18.0
-	k8s.io/component-base v0.18.0
+	k8s.io/api v0.19.0-rc.2
+	k8s.io/apimachinery v0.19.0-rc.2
+	k8s.io/client-go v0.19.0-rc.2
+	k8s.io/code-generator v0.19.0-rc.2
+	k8s.io/component-base v0.19.0-rc.2
```

Comment:
I'll merge this.

Can you submit another PR to move klog from v1 to v2?  Like this one: https://github.com/kubernetes-csi/external-attacher/pull/238/files

Reply:
This has been submitted at https://github.com/kubernetes-csi/external-snapshotter/pull/340 .